<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFrame" name="CircularProgress">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>248</width>
        <height>248</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius: 124px;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.249 rgba(31, 244, 75, 239), stop:0.25 rgba(0, 0, 0, 89));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>7</number>
       </property>
       <property name="topMargin">
        <number>7</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>7</number>
       </property>
       <item>
        <widget class="QFrame" name="TitleFrame">
         <property name="styleSheet">
          <string notr="true">QFrame{
border-radius:117px;
	
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.491525 rgba(93, 93, 93, 255), stop:0.80226 rgba(73, 73, 73, 255), stop:0.949153 rgba(45, 45, 45, 255), stop:0.988701 rgba(0, 0, 0, 255));

	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.0603015 rgba(48, 48, 107, 255), stop:0.959799 rgba(29, 29, 76, 255), stop:1 rgba(0, 0, 75, 255));
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>25</number>
          </property>
          <property name="topMargin">
           <number>48</number>
          </property>
          <property name="rightMargin">
           <number>21</number>
          </property>
          <property name="bottomMargin">
           <number>26</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>11</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="loadingLabel">
              <property name="styleSheet">
               <string notr="true">QLabel{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	color: rgb(213, 213, 213);

}</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;L&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;oading...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="percentLabel">
              <property name="styleSheet">
               <string notr="true">QLabel{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	color: rgb(244, 244, 244);
}</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
